osu.Game.Tests/Audio/TestSceneFilter.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using ManagedBass.Fx;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Graphics;
+using osu.Game.Audio.Effects;
+using osu.Game.Beatmaps;
+using osu.Game.Tests.Visual;
+
+namespace osu.Game.Tests.Audio
+{
+    public class TestSceneFilter : OsuTestScene
+    {
+        [Resolved]
+        private AudioManager audio { get; set; }
+
+        private WorkingBeatmap testBeatmap;
+        private Filter lowPassFilter;
+        private Filter highPassFilter;
+        private Filter bandPassFilter;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            testBeatmap = new WaveformTestBeatmap(audio);
+            AddRange(new Drawable[]
+            {
+                lowPassFilter = new Filter(audio.TrackMixer)
+                {
+                    FilterType = BQFType.LowPass,
+                    SweepCutoffStart = 2000,
```

Comment:
Is there a reason these start at such a low value? Wouldn't there be a sudden jump down from the full range to this value? I would've thought the default cutoff would be set to either 0 or 22050 and then transformed from there.